gcc/testsuite/rust.test/xfail_compile/type-alias1.rs

```diff
@@ -0,0 +1,6 @@
+type TypeAlias = (i32, u32);
+
+fn main() {
+    let a: TypeAlias;
+    a = (123, 456f32); // { dg-error "expected .u32. got .f32." }
```

Comment:
@dkm is this the correct way to do this the error message is:

```
test.rs:5:15: error: expected [u32] got [f32]
    5 |     a = (123, 456f32); // { dg-error "expected .u32. got .f32." }
      |
```

Reply:
Looks correct yes. You'll quickly see if it works correctly as the test will FAIL if the regex does not match an error message at this exact line !